### ISSUE TITLE
React@15.3.0 compatibility

### ISSUE THREAD

**Issue by necolas** [necolas/react-native-web#178]
React@15.3.0 introduced a new warning https://github.com/facebook/react/pull/7132

React Native's `StyleSheetPropType` relies on the deprecated behaviour. Will fix once RN has decided how it wants to fix the warnings.

**Comment by grigio** [comment]
I think it is related. With `create-react-app` I get tons of warning/errors but it depends on react `^15.2.1`
```
warning.js:44 Warning: You are manually calling a React.PropTypes validation function for the `visibility` prop on `Image`. This is deprecated and will not work in the next major version. You may be seeing this warning due to a third-party PropTypes library. See https://fb.me/react-warning-dont-call-proptypes for details.
```

**Comment by necolas** [comment]
That's because `^15.2.1` will install `15.3.0` if available.

**Comment by grigio** [comment]
Yeah I noticed it, thanks. So also `create-react-app` is a nice starter kit for RNW you just have to "eject" the app and add this line https://github.com/grigio/react-native-plus-web-boilerplate/blob/master/config/webpack.config.dev.js#L34

**Comment by necolas** [comment]
Fixed in 2291544373d51b771117783339dc498b126eba76 and 0.0.42
